find_broken_links.py

```diff
@@ -0,0 +1,118 @@
+import re
+import socket
+import sys
+import urllib.request
+from glob import iglob
+from typing import Iterator, TextIO, Tuple
+from urllib.error import HTTPError, URLError
+
+from dataclasses import dataclass
+
+REQUEST_HEADERS = {'User-Agent': 'https://github.com/RTBHOUSE/here-be-pythons'}
```

Comment:
I think request headers are not good candidate for a constant. Maybe USER_AGENT_HEADER, but only if it's use for auth or other significant purposes.